python/doc/math_notations.sty

```diff
@@ -6,12 +6,59 @@
 
 % The physical model
 \newcommand{\model}{g}
+\newcommand{\inputDim}{d}  % The input dimension of the model
+\newcommand{\outputDim}{d_Y}  % The output dimension of the model
+\newcommand{\inputDomain}{\mathcal{D}}  % Input domain of the model
+\newcommand{\metaModel}{\widetilde{\model}}  % The surrogate model
+
+% Probabilistic modeling
+\newcommand{\inputRV}{\vect{X}}  % The input random vector of the model
+\newcommand{\outputRV}{\vect{Y}}  % The output random vector of the model
+\newcommand{\inputMeasure}{\mu_{\inputRV}}  % The distribution of the input random vector of the model
+\newcommand{\outputMeasure}{\mu_{\outputRV}}  % The distribution of the output random vector of the model
+% Realization of the random vetor
+\newcommand{\inputReal}{\vect{x}}  % An observation of the input random vector of the model
+\newcommand{\outputReal}{\vect{y}}  % An observation of the output random vector of the model
+\newcommand{\RVZ}{\vect{Z}}
```

Comment:
The name of the macro reflects the mathematical meaning. Its content reflects the actual letter. Using the content of the macro inside the name of the macro removes a part of the interest of using a macro. 
```suggestion
\newcommand{\standardRV}{\vect{Z}}
```